test/middlewares/permissions.unit.test.js

```diff
@@ -230,6 +229,19 @@ describe('middlewares > permissions', () => {
         smart_action_id: 'users-action@@@bulk',
       };
 
+      const getDependencies = () => ({
+        ...defaultDependencies,
+        modelsManager: {
+          getModelByName: jest.fn().mockReturnValue({ name: 'users' }),
+        },
+        configStore: {
+          Implementation: {
+            getModelName: jest.fn().mockReturnValue('users'),
+            ResourcesGetter: jest.fn().mockImplementation(() => ({ count: () => 3 })),
+          },
+        },
+      });
```

Comment:
[MISC] Why have you made this a function and not a static obj?

Reply:
so that the stub counters reset to zero